src/odb/test/cpp/sky130_test_fixture.h

```diff
@@ -25,8 +30,10 @@ class Sky130TestFixutre : public ::testing::Test
     odb::lefin lef_reader(
         db_.get(), &logger_, /*ignore_non_routing_layers=*/false);
     lib_ = OdbUniquePtr<odb::dbLib>(
-        lef_reader.createTechAndLib(
-            "sky130", "sky130", "data/sky130hd/sky130_fd_sc_hd.tlef"),
+        lef_reader.createTechAndLib(  //
```

Comment:
Is there a purpose to the `//` ?

Reply:
yes, it forces `clang-format` to format the way I want it to. Without that, it would put the macro in one line and the string literal into another line.
Adding EOL comments like this is a good way to 'help' clang-format do the right thing.

Reply:
Perhaps
```
    dbLib* lib = lef_reader.createTechAndLib(
        "sky130", "sky130", DATA_PREFIX "data/sky130hd/sky130_fd_sc_hd.tlef");
    lib_ = OdbUniquePtr<odb::dbLib>(lib, &odb::dbLib::destroy);
```
avoid the wrapping issue and is a bit clearer imho.

Reply:
well, but that is not what clang-format would do. So we have to force its hand a bit.

Reply:
That is what it would do as I formatted it before sending it.